### PR TITLE
Updates Read-Me File

### PR DESCRIPTION
**Pull request by arlene-perez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,39 @@
...
+
+### 💡💥❓ How things work
+
+Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the issue directly in the master branch or creating a pull request, I create a new branch that is call something like first-timers-only-typo-in-title. GitHub will notify the "first-timers-only issue bot" about the new branch using webhooks. Currently, the bot will see that a new branch was created and it will create a new issue at https://github.com/hoodiehq/camp/issues/new based on a template and assign the first-timers-only and the up-for-grabs label. If the commit body can be used to add some context information. If left empty, the 🤔 What you will need to know section of the issue will simply say "Nothing :)". Otherwise the commit body will be used there.
```
Comment (arlene-perez) [inline README.md]: Thank you for that! Completely missed it :)

File: README.md [hunk 2]
```diff
@@ -1,8 +1,39 @@
...
+We are in the progress of updating the bot to watch only for branches starting with "first-timers-only".😃
+
+NOTE: Add picture of an issue example
+
+### ✅ ➕ Requirements
+1) You'll need an Access Token from Github. Follow steps 1-9 from this [article](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) to generate your token. ⚠️ Make sure to SAVE the token in a document/editor file because it will only be shown to you once.
+
+2) Install [Node.js](https://nodejs.org/en/)
+
+3) Install [Hapi](https://hapijs.com/)
```
Comment (arlene-perez) [inline README.md]: Leave out the requirements section?

File: README.md [hunk 3]
```diff
@@ -1,8 +1,39 @@
...
+
+### 💡💥❓ How things work
+
+Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the issue directly in the master branch or creating a pull request, I create a new branch that is call something like first-timers-only-typo-in-title. GitHub will notify the "first-timers-only issue bot" about the new branch using webhooks. Currently, the bot will see that a new branch was created and it will create a new issue at https://github.com/hoodiehq/camp/issues/new based on a template and assign the first-timers-only and the up-for-grabs label. If the commit body can be used to add some context information. If left empty, the 🤔 What you will need to know section of the issue will simply say "Nothing :)". Otherwise the commit body will be used there.
+
+We are in the progress of updating the bot to watch only for branches starting with "first-timers-only".😃
+
+NOTE: Add picture of an issue example
```
Comment (arlene-perez) [inline README.md]: Would we take a pic of the issue created?

File: README.md (hunk 2 above)
Comment (gr2m) [inline README.md]: yes

File: README.md (hunk 3 above)
Comment (gr2m) [inline README.md]: yeah, open an issue that looks good in your browser than take a screenshot of the browser. On Mac you can press command + shift + 4 then space to take a screenshot of a single application

**Pull request merged**